src/test/java/com/rocketden/main/service/RoomServiceTests.java

```diff
@@ -369,12 +369,11 @@ public void updateRoomSettingsInvalidDuration() {
 
         UpdateSettingsRequest request = new UpdateSettingsRequest();
         request.setInitiator(UserMapper.toDto(host));
+        request.setNumProblems(-1);
 
-        request.setDifficulty(ProblemDifficulty.EASY);
-        request.setDuration(-1L);
-
-        ApiException exception = assertThrows(ApiException.class, () -> roomService.updateRoomSettings(ROOM_ID, request));
-        assertEquals(TimerError.INVALID_DURATION, exception.getError());
+        ApiException exception = assertThrows(ApiException.class, () ->
+                roomService.updateRoomSettings(ROOM_ID, request));
+        assertEquals(ProblemError.INVALID_NUMBER_REQUEST, exception.getError());
```

Comment:
nit: Change this back (I also had this issue when fixing giant merge conflicts lol)

Reply:
Ah thank you for catching this 👍 

Changed back!